mypy.ini

```diff
@@ -0,0 +1,6 @@
+[mypy]
+ignore_missing_imports = True
```

Comment:
@uniqueg I am left with these errors for now, since most are dependent on `foca` how do I go by slowing them supressing `attr-defined` globally doesn't seem to be right and I can't define this in `mypy.ini` all i can do is `# type: ignore`, should I do crazy with `# type: ignore` for these errors?

```
pro_wes/ga4gh/wes/service_info.py:33: error: "Config" has no attribute "foca"  [attr-defined]
pro_wes/app.py:32: error: "Config" has no attribute "foca"  [attr-defined]
pro_wes/app.py:39: error: "Config" has no attribute "foca"  [attr-defined]
pro_wes/ga4gh/wes/models.py:161: error: "str" has no attribute "name"  [attr-defined]
pro_wes/tasks/track_run_progress.py:60: error: "Config" has no attribute "foca"  [attr-defined]
pro_wes/tasks/track_run_progress.py:98: error: "RunLog" has no attribute "pop"  [attr-defined]
pro_wes/tasks/track_run_progress.py:108: error: "dict[Any, Any]" has no attribute "polling_wait"  [attr-defined]
pro_wes/tasks/track_run_progress.py:120: error: "dict[Any, Any]" has no attribute "polling_attempts"  [attr-defined]
pro_wes/tasks/track_run_progress.py:127: error: "dict[Any, Any]" has no attribute "polling_attempts"  [attr-defined]
pro_wes/tasks/track_run_progress.py:151: error: "RunLog" has no attribute "pop"  [attr-defined]
pro_wes/ga4gh/wes/workflow_runs.py:59: error: "Config" has no attribute "foca"  [attr-defined]
```

Reply:
Yes, I'm afraid so - it's what we did in the other repos as well for now.

We will fix this with the upcoming FOCA release, but it will take a couple of weeks. Then we can revisit.